default.nix

```
@@ -313,7 +313,7 @@ rec {
     };
 
     testDerivationDeps =
-      (with nixpkgs; [ wabt bash perl getconf moreutils nodejs-16_x sources.esm ]) ++
+      (with nixpkgs; [ wabt bash perl getconf moreutils nodejs-16_x ]) ++ # TODO: sources.esm
```

Unclear why we needed this, but it seems we don’t need it any more.
```suggestion
      (with nixpkgs; [ wabt bash perl getconf moreutils nodejs-16_x ]) ++
```